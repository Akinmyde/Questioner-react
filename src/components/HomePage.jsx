import React from 'react';
import { Link } from 'react-router-dom';

const HomePage = () => (
  <div id="banner">
    <div>
      <h1>Welcome to Questioner App</h1>
      <div className="banner-button">
        <Link className="md" to="/login">Get Started</Link>
      </div>
    </div>
  </div>
);

export default HomePage;
