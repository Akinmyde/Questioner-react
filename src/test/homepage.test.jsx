import 'babel-polyfill';
import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../components/presentations/HomePage/HomePage';

describe('Login Page', () => {
  it('should render login correctly', () => {
    const homepage = shallow(<homepage />);
    expect(homepage).toMatchSnapshot();
  });
});
