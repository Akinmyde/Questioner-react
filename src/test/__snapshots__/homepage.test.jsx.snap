// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page should render login correctly 1`] = `<homepage />`;
